ChangeLog

```diff
@@ -2,7 +2,7 @@
 #---------------------------------------------------------------------------
 3.42 2020-11-10 Karl Williamson <khw@cpan.org>
 	- Drop EUMM version prereq, Graham Knop++
-3.41 2020-10-09 Karl Williamson <khw@cpan.org>
+3.42 2020-10-09 Karl Williamson <khw@cpan.org>
```

Comment:
This looks like a misapplied search-and-replace. But I'm not sure why this commit is here at all.

Reply:
I believe this has now been corrected.